platforms/windows/SQLitePlugin/SQLitePlugin.cpp

```diff
@@ -0,0 +1,511 @@
+#include "pch.h"
+
+#include "SQLitePlugin.h"
+
+using namespace winrt;
+using namespace winrt::Microsoft::ReactNative;
+using namespace winrt::Windows::Foundation;
+using namespace winrt::Windows::Storage;
+using namespace winrt::Windows::Storage::Streams;
+using namespace winrt::Windows::Security::Cryptography;
+
+namespace SQLitePlugin
+{
+    void SQLitePlugin::Attach(
+        JSValue options,
+        std::function<void(std::string)> onSuccess,
+        std::function<void(std::string)> onFailure) noexcept
+    {
+        SerialReactDispatcher.Post(
+            [onFailure{ std::move(onFailure) }]()
```

Comment:
What are you trying to do here? Is there a simpler way to capture onFailure?

Reply:
I am moving the callbacks into a lambda that goes into the dispatcher queue and waits its turn to get executed.